### DISCUSSION TITLE
Conflicts with dependencies Docker Bat file?

### DISCUSSION THREAD

**Discussion by Necromnomicon** [joshreve/dactyl-keyboard#77] [Q&A]
Hello. I had things running well when I set things up through the auto docker install back in December. I updated docker the other day and started having problems with Docker not running. I ended up having to do a complete uninstall, and purged all of dockers hidden files/registry entries. 
After re-setting up Docker. I downloaded the latest master branch. I tried to run the build_docker.bat file and on step 4/8 RUN conda env create -f environment.yml  I get a bunch of conflicts, and the process ultimately fails.  Any insight as to why? I'm not very savvy in regards to these things, so if you need any additional info, please let me know, and I'll do my best to provide it. Thanks.

**Comment by wcunning** [comment]
I ran into the same issue installing on Linux. The errors in question are in the attached errors.txt file. 
[errors.txt](https://github.com/joshreve/dactyl-keyboard/files/8104359/errors.txt)

**Reply by wcunning** [reply]
I ended up getting things running with Conda instead of Docker.

**Reply by Necromnomicon** [reply]
Much appreciation for the info, thanks.  I plan on giving Conda a go when I have the time to sit down with it.

**Reply by joshreve** [reply]
Glad you got it working.  I feel like the dependencies keep changing.  I think I've fixed this a few times.  The cadquery seems very brittle in this area.  I'm testing it out again and can hopefully fix it. . . again.
